ads/google/csa.js

```diff
@@ -212,15 +212,24 @@ function requestCsaAds(global, data, afsP, afsA, afshP, afshA) {
  */
 function getAdType(data) {
   if (data['afsPageOptions'] != null && data['afshPageOptions'] == null) {
+    validateData(data, ['afsPageOptions', 'afsAdblockOptions'], []);
```

Comment:
will this be a breaking change for existing mis-configured users? are we OK with it?
or, at least, to mitigate the breakage, shall we do:
```
validateData(data, ['afsPageOptions', 'afsAdblockOptions']);
```